packages/mdc-select/README.md

```diff
@@ -358,3 +384,25 @@ selects.native.addEventListener('change', changeHandler);
 ```
 
 We are looking into building this functionality into `MDCSelect` in the future.
+
+## Multi Select
+
+Currently MDC Web supports the CSS only implementation of Multi Select. Add the `multple` attribute
```

Comment:
nit: I think we can change the wording here to just "MDC-Web  implements multi-select on top of the `<select multiple>` element.